### DISCUSSION TITLE
How can I start contributing? What is the "good first issue" on this project?

### DISCUSSION THREAD

**Discussion by ghost** [pavlovcik/uad-ui#397] [Q&A] [answered]
Hi
I'm going to contribute to this opensource project.
I'm new to this project and please give me some issues to solve for the first time.

**Comment by 0x4007** [comment, marked as answer]
We don't choose issues for bounty hunters because we don't know what level of expertise bounty hunters have, or what type of problems are interesting to each bounty hunter. 

Generally speaking, you will have a smoother experience starting out with the short, tightly scoped, and well specified bounties. For example, bounties that are 100 or 200 USD. These generally can be handled within a few hours and are quick to review and provide payments for. 

I have observed that the more expensive bounties tend to be quite a bit more complex to deal with, which can be interesting to those who are already familiar with the project and looking for a multi-day challenge. These generally end up taking longer not only to solve, but to be reviewed and to be paid for.
